locale/fi/LC_MESSAGES/django.po

```diff
@@ -526,6 +526,12 @@ msgstr "Virheellinen pk \"{pk_value}\" - kohdetta ei ole olemassa."
 msgid "Invalid value in filter %(filter)s"
 msgstr "Virheellinen arvo filtterissä %(filter)s"
 
+msgid "You need to login with strong authentication to reserve this resource."
+msgstr ""
```

Comment:
"Sinun täytyy kirjautua vahvalla tunnistautumistavalla varataksesi tämän resurssin"

Reply:
Also these messages are not very user-friendly. Maybe the login methods they should use should be included in the message? Just an idea for improvement, could be a follow-up ticket to this one.

Reply:
I didn't change it here. I will update it in follow up ticket.